examples/audio_player_interaction/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '11.0'
+# platform :ios, '12.0'
```

Comment:
was this iOS upgrade intended?

Reply:
Probably not. When I generated the macOS project in the example it updated the platform numbers. DO you want me to revert it?

Reply:
I personally think it's fine, unless @sowens-csd has reasons not to